src/QuoteBox.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
+import './App.css';
+
+
+class QuoteBox extends Component {
+  handleClick(e) {
+    e.preventDefault();
+    this.forceUpdate();
+  }
+  render() {
+    const quotes = [{
```

Comment:
Don't handle data in the component
It is just supposed to render stuff

Pass them down as `props` with 
```<QuoteBox  quotes={quotes} />```

Then you can access it with `this.props.quotes`